Dockerfile-ARM64v8

```diff
@@ -0,0 +1,24 @@
+FROM arm64v8/golang:1.15
```

Comment:
Changed from to be arm64v8 based.

Reply:
Hey! Could you change this to `1.15-alpine` - the `1.15` default is buster.